_posts/en/newsletters/2024-12-20-newsletter.md

```diff
@@ -0,0 +1,1588 @@
+to prevent the spender from learning the identity of the receiver's
+node.
+
+{:#pooledmining}
+A [new mining interface][news325 mining] for Bitcoin Core saw
+development with the goal of supporting miners using the [Stratum v2][topic
+pooled mining] protocol that can be set to allow each miner to select
+their own transactions.  However, Anthony Towns noted earlier in the
+year that independent transaction selection could [raise share validation
+costs][news315 shares] for mining pools.  If those pools responded by
+limiting validation, it could allow an invalid shares attack that's
+similar to the well-known [block withholding attack][topic block
+withholding].  A 2011 proposed solution to withholding attacks was
+discussed, although it would require a challenging consensus change.
+
+<div markdown="1" class="callout" id="releases">
+### Summary 2024: Major releases of popular infrastructure projects
+
+- [LDK 0.0.119][] added support receiving payments to multi-hop [blinded
+  paths][topic rv routing].
+
+- [HWI 2.4.0][] added support for Trezor Safe 3.
+
+- [Core Lightning 24.02][] included improvements to the `recover` plugin
+  that "make emergency recoveries less stressful", improvements to
+  [anchor channels][topic anchor outputs], 50% faster block chain
+  syncing, and a bug fix for the parsing of a large transaction found on
+  testnet.
+
+- [Eclair v0.10.0][] "added official support for the [dual-funding
+  feature][topic dual funding], an up-to-date implementation of BOLT12
+  [offers][topic offers], and a fully working [splicing][topic splicing]
+  prototype".
+
+- [Bitcoin Core 27.0][] deprecated libbitcoinconsensus, enabled [v2
+  encrypted P2P transport][topic v2 p2p transport] by default, allowed
+  the use of opt-in topologically restricted until confirmation
+  ([TRUC][topic v3 transaction relay]) transaction policy on test
+  networks, and added a new [coin selection][topic coin selection]
+  strategy to be used during high feerates.
+
+- [BTCPay Server 1.13.1][] (and previous releases) made webhooks more
+  extensible, added support for [BIP129][] multisig wallet import,
+  improved plugin flexibility and begun migrating all altcoin support to
+  plugins, and added support for BBQr-encoded [PSBTs][topic psbt].
+
+- [Bitcoin Inquisition 25.2][] added support for [OP_CAT][topic op_cat]
+  on signet.
+
+- [Libsecp256k1 v0.5.0][] sped up key generation and signing and reduces
+  the compiled size "which [its developers] expected to benefit embedded
+  users in particular."
+
+- [LDK v0.0.123][] included an update to its settings for [trimmed
+  HTLCs][topic trimmed htlc] and improvements to [offers][topic offers]
+  support.
+
+- [Bitcoin Inquisition 27.0][] added signet enforcement of [OP_CAT][] as
+  specified in [BIN24-1][] and [BIP347][].  It also included "a new
+  `evalscript` subcommand for `bitcoin-util` that can be used to test
+  script opcode behavior".  Support was dropped for `annexdatacarrier`
```

Comment:
```suggestion
  script opcode behavior".  The release dropped support for `annexdatacarrier`
```

Reply:
Leaving as-is.  Passive is more efficient here.